README.md

```diff
@@ -249,6 +249,11 @@ If your project is on GitHub, you could take advantage of GitHub Actions and the
 usethis::use_github_action("lint-project")
 ```
 
+#### super-linter ####
+
+[Super-Linter](https://github.com/github/super-linter) and [MegaLinter](https://megalinter.github.io/latest/),
+which performs linting for various files, includes lintr for R.
+They execute the `lintr::lint()` function on the `.r` or `.rmd` files included in the project.
```

Comment:
case insensitive?

Reply:
It does not seem to be case sensitive. Is there a better way to describe it?

Reply:
"R and R Markdown files" ?

Reply:
that sounds better yes